test/inductor/test_decompose_mem_bound_mm.py

```diff
@@ -117,6 +118,29 @@ def test_decompose_bmm(self, b, m, n, k, should_decompose):
         )
         counters.clear()
 
+    @parametrize(
```

Comment:
Hi, @hl475  @eellison  actually this case is not tested because the the test class is decorated as @requires_gpu, and if I run this case in machine that has gpu, it fails. So I suggest refine the case.